packages/frontend/src/utils/wallet.ts

```diff
@@ -711,40 +736,45 @@ export default class Wallet {
     async checkLinkdropInfo(fundingContract, fundingKey) {
         const account = await this.getAccount(fundingContract);
 
-        const contract = new nearApiJs.Contract(account, fundingContract, {
+        const contract = new LinkDropContract(account, fundingContract, {
+            changeMethods: [],
             viewMethods: ['get_key_balance', 'get_key_information'],
```

Comment:
Is it okay to not pass sender here? I see the original code passed it in.

Reply:
![Screenshot 2023-08-05 at 12 53 16 AM](https://github.com/mynearwallet/my-near-wallet/assets/125173477/a8ba9ddd-7e09-4488-84da-79129bcde4f7)

ContractMethod should not have a key named "sender"